sites/docs/src/content/docs/tutorials/google_colab/nf-core-colab-guide.md

```diff
@@ -0,0 +1,162 @@
+---
+title: "Running nf-core Pipelines on Google Colab"
+subtitle: "A guide to running and interacting with nf-core pipelines using Colab and VS Code"
+---
+
+# Introduction
+
+Running nf-core pipelines can be computationally intensive, requiring resources not easily available to students, newcomers, or participants in hands-on training workshops.
+This tutorial shows two ways you can use Google Colab to run nf-core pipelines entirely in the cloud, either in a browser-based execution or connected to VS Code via the vscode-colab package for a full development experience.
+While Colab has limitations, such as session timeouts and a lack of root access, it offers a free and accessible platform ideal for learning, teaching, and prototyping workflows in resource-constrained environments.
+Also make sure to checkout the [blogpost](https://nf-co.re/blog/2025/nf-core-colab-guide) on this topic for deeper insights and more tips for running nf-core pipelines using Colab.
+
+# Setting up the environment in Google Colab
+
+After opening a Colab notebook with the machine type of your choice, the first step is to install Nextflow and it's dependencies. Since Colab does not provide root access, you will need to install Nextflow in the user's home directory.
+
+After opening a Colab notebook with the machine type of your choice, the first step is to install Nextflow and its dependencies. Since Colab does not provide root access, you will need to install Nextflow in your home directory.
+
+Before installing Nextflow, you must install Java. In Colab, run the following commands in a code cell using the `%` and `!` prefixes, or run them directly in your terminal (without the prefixes).
+
+```python
+%cd .. # use '%'before cd to make a permanent change of directories
+!sudo apt update
+!sudo apt install openjdk-17-jdk
+!export JAVA_HOME=/usr/lib/jvm/java-17-openjdk-amd64
+!export PATH=$JAVA_HOME/bin:$PATH
+!source ~/.bashrc
+```
+
+Next, install Nextflow and make it executable by running the following commands in a code cell as before:
+
+```python
+!wget -qO- https://get.nextflow.io | bash # Download Nextflow
+!mv nextflow /usr/bin/nextflow # Move to a path Colab can access
+!chmod +x /usr/bin/nextflow # Make it executable
+!nextflow -v # Test it
+```
+
+Due to a lack of root access, it is not possible to run nf-core, or any Nextflow pipelines, using the `-profile docker` or `-profile singularity` container based configuration profiles.
+
+That leaves the conda profile as the only way to run pipelines.
+However, as Google Colab does not support native conda functionality, you need to install the [condacolab](https://pypi.org/project/condacolab/) package, which will set up Conda for you by running the following commands:
```

Comment:
```suggestion
However, as Google Colab does not support native Conda functionality, you need to install the [condacolab](https://pypi.org/project/condacolab/) package, which will set up Conda for you by running the following terminal and notebook commands:
```

If I understand correctly? I don't fully understand why the CLI ocmmand has a `!` and then you switch to directly writing pyton code

Reply:
Is it normal that in Google Colab notebooks that the primary language of the code blocks are python, or is this something you configured?